client/src/wi_stuff.cpp

```diff
@@ -1135,10 +1135,11 @@ void WI_drawNetgameStats()
 
 		unsigned int x = NG_STATSX;
 		// [RH] Only use one graphic for the face backgrounds
-		if (demoplayback)
+		//enaiel: Fix incorrect player background when showing old intermission
```

Comment:
If your change removes old code, please just delete the old code instead of commenting it out. The old code is preserved in git history so if it's needed later, it can be found.

Reply:
I removed this in preparation to merge